src/classes/CRLP_RollupProcessor.cls

```diff
@@ -543,6 +553,25 @@ public inherited sharing class CRLP_RollupProcessor {
         }
     }
 
+    /**
+     * @description Set needsUpdate and CustomizableRollups_UseSkewMode__c true if >= Skew Mode Threshold
+     * @param updatedRecord Update SObject record for rollup totals
+     * @param needsUpdate Boolean indicating whether or not record requires update
+     * @return Boolean
+     */
+    private Boolean needsSkewModeOverrideSet(SObject updatedRecord, Boolean needsUpdate) {
+        if (((Decimal) updatedRecord.get('npo02__NumberOfClosedOpps__c') >= maxRelatedOppsForNonSkewMode ||
+            (Decimal) updatedRecord.get('npo02__NumberOfMembershipOpps__c') >= maxRelatedOppsForNonSkewMode) ||
+            (updatedRecord.getSObjectType() == Contact.SObjectType) &&
+            (Decimal) updatedRecord.get(UTIL_Namespace.StrTokenNSPrefix('Number_of_Soft_Credits__c')) >= maxRelatedOppsForNonSkewMode)
```

Comment:
I think there's an extra closing parenthesis here after Contact.SobjectType. I belivee the intent is to wrap the entire && condition in a parenthesis block:
```suggestion
            (updatedRecord.getSObjectType() == Contact.SObjectType &&
            (Decimal) updatedRecord.get(UTIL_Namespace.StrTokenNSPrefix('Number_of_Soft_Credits__c')) >= maxRelatedOppsForNonSkewMode))
```

Reply:
Eek!!!  Okay, so that begs the question, is this test too difficult to read?  Should I refactor this?